Cimpress.Stereotype/MaterializationResponse.cs

```diff
@@ -11,31 +12,32 @@ namespace Cimpress.Stereotype
     public class MaterializationResponse : IMaterializationResponse
     {
         private readonly string _accessToken;
-        private readonly ILogger _logger;
+        private readonly byte[] _rawResponse;
+        private readonly ILogger<StereotypeClient> _logger;
         private readonly IRestClient _restClient;
 
-        public MaterializationResponse(string accessToken, Uri uri, ILogger logger, IRestClient restClient)
+        public MaterializationResponse(string accessToken, Uri uri, byte[] rawResponse, ILogger<StereotypeClient> logger, IRestClient restClient)
         {
             Uri = uri;
+            _rawResponse = rawResponse;
             _accessToken = accessToken;
             _logger = logger;
             _restClient = restClient;
         }
-        
-        public MaterializationResponse(string accessToken, Uri uri, ILogger logger) : this(accessToken, uri, logger, new RestClient())
```

Comment:
It is a public class, so maybe we could leave this constructor, that would provide null `rawResponse` ? Especially once we will be implementing the async scenario, this will be useful.

Reply:
I put it back.